rest/data/build_baron.go

```diff
@@ -29,7 +28,7 @@ type FailingTaskData struct {
 // BbFileTicket creates a JIRA ticket for a task with the given test failures.
```

Comment:
Can you please add a link to the fake BF that you generated when testing this?

Reply:
thanks! and did you generate it by clicking the file ticket button on the first execution of the task?

Reply:
Yes, I generated one with the latest execution, one with the first execution
I did this with a couple of different tests as well and checked the URL to make sure the execution is the same